### ISSUE TITLE
Attempted to retain deallocated object

### ISSUE THREAD

**Issue by rasantiago** [apollographql/apollo-ios#135]
I am hitting an error within FetchQueryOperation.  It is complaining that it is trying to access a deallocated object.  Specifically, it complains about the following line found in the `start` method:

```  
override public func start() {
...    
    client.store.load(query: query) { (result, error) in
...
```

The specific error is reported as:

```
libswiftCore.dylib`_swift_abortRetainUnowned:
    0x1082df6e0 <+0>:  leaq   0x30b91(%rip), %rax       ; "attempted to retain deallocated object"
    0x1082df6e7 <+7>:  movq   %rax, 0x9c83a(%rip)       ; gCRAnnotations + 8
    0x1082df6ee <+14>: int3   
->  0x1082df6ef <+15>: nop    
```

I am new to using the Apollo iOS client so am only trying to run a very simple example against a vanilla GraphQL server.  My query is as follows:

```
query upcCls {
    upcomingClasses {
        id
    }
}
```
I have verified this query works and returns results using direct post (via Postman and CURL).  My Swift code is as follows:

```
let apollo = ApolloClient(url: URL(string: "<myGraphQLServerUrl>")!)
let q = UpcClsQuery()
apollo.fetch(query: q) { (result, error) in
    guard let data = result?.data else {
        print(error!)
        return
    }
    print(data)
}
```

Also, using Xcode 8.3.3 and Apollo Cocoapod version 0.6.2, using Node 6.10.3 to run Apollo-Codegen 0.16.5

Lastly, if use a cache policy of `.fetchIgnoringCacheData` on the fetch then the above code works.

Thanks in advance for any help.

**Comment by martijnwalraven** [comment]
I'm not sure if this is related to https://github.com/apollographql/apollo-ios/issues/133, but could you give 0.6.3 a try and see if that fixes your issue?

**Comment by rasantiago** [comment]
No luck.  Tried 0.6.3 and 0.6.4.  Same error.  But, if I change the variable declaration for `client` in `FetchQueryOperation` from `unowned let client: ApolloClient` to `let client: ApolloClient` all seems to work fine.

**Comment by martijnwalraven** [comment]
Ah, that makes sense. The assumption is that an `ApolloClient` instance will be kept around, by storing it in a global variable for example. In most cases, that's what you want, because clients are relatively expensive to create, they set up a store, etc. But you should still be able to create a one-off client for experimentation, so the operation should retain the client. This should be fixed in 0.6.5.
